tests/gold_tests/tls_hooks/tls_hooks16.test.py

```diff
@@ -70,9 +69,7 @@
 
 ts.Streams.stderr = "gold/ts-client-hello-1.gold"
 
-snistring = "Client Hello callback 0"
-ts.Streams.All = Testers.ContainsExpression(
-    "\A(?:(?!{0}).)*{0}(?!.*{0}).*\Z".format(snistring), "Client Hello message appears only once", reflags=re.S | re.M)
+#snistring = "Client Hello callback 0"
```

Comment:
Was this intended to be commited? If so, can it just be removed?

Reply:
done